package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-lessons",
+  "version": "1.0.0",
+  "description": "React training lessons",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack-dev-server --open --config ./webpack.config.dev.js --mode development",
+    "prod": "webpack --config ./webpack.config.prod.js --mode production"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/itsikal/react-trainings.git"
+  },
+  "author": "Igor Tsikal",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/itsikal/react-trainings/issues"
+  },
+  "homepage": "https://github.com/itsikal/react-trainings#readme",
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "eslint": "^5.4.0",
+    "html-webpack-plugin": "^3.2.0",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2",
+    "uglifyjs-webpack-plugin": "^1.3.0",
+    "webpack": "^4.17.1",
+    "webpack-cli": "^3.1.0",
+    "webpack-dev-server": "^3.1.6"
```

Comment:
By my own experience, I don't recommend you using of tilde and caret for version definition. Because, updating of dependencies can broke your project.
The best practices is manual updating of dependencies